clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -11164,12 +11164,13 @@ def err_ext_int_max_size : Error<"%select{signed|unsigned}0 _ExtInt of bit "
 def err_esimd_glob_cant_init : Error<
   "SYCL explicit SIMD does not permit private global variable to have an initializer">;
 
-def err_sycl_kernel_incorrectly_named : Error<"%0 is an invalid kernel name type">;
-def note_invalid_type_in_sycl_kernel : Note<
+def err_nullptr_t_type_in_sycl_kernel : Error<"'std::nullptr_t' is an invalid kernel name ">;
+def err_invalid_std_type_in_sycl_kernel : Error<"%0 is an invalid kernel name, %q1 is declared in the \'std\' namespace ">;
```

Comment:
80-col limit, also, you don't need to escape the single quotes in the string literal.

Reply:
This was the final text we decided upon keeping in mind the 80-col limit.
I did get rid of the escaping.

Reply:
What he means is that the file itself has an 80-column limit.  So you should put the error message on multiple lines (see the rest of the file for examples for how these are done).